lib/protobuf/builder.ex

```diff
@@ -5,7 +5,7 @@ defmodule Protobuf.Builder do
 
   @spec new(module) :: %{required(:__struct__) => module} when module: module()
   def new(mod) when is_atom(mod) do
-    mod.__default_struct__()
+    struct(mod, [])
```

Comment:
nit: I think this should also work:

```suggestion
    struct(mod)
```

Reply:
Noice, missed that `[]` is the default! 😉